src/gui/uithememanager.cpp

```diff
@@ -143,21 +146,14 @@ QColor UIThemeManager::getColor(const QString &id, const QColor &defaultColor) c
 
 QString UIThemeManager::getIconPath(const QString &iconId) const
 {
-#if (defined(Q_OS_UNIX) && !defined(Q_OS_MACOS))
```

Comment:
Why did you remove this code?

Reply:
before this PR we never called this method(`UIThemeManager::getIconPath`) but `IconProvider::getIconPath`, which doesn't have this code, so this code was actually never used.